packages/esm-patient-tests-app/src/test-results/overview/common-overview.component.tsx

```diff
@@ -10,55 +10,20 @@ import styles from './common-overview.scss';
 
 const DashboardResultsCount = 5;
 
-interface CommonOverviewPropsBase {
-  overviewData: Array<OverviewPanelEntry>;
-  insertSeparator?: boolean;
+interface CommonOverviewProps {
   patientUuid?: string;
+  overviewData: Array<OverviewPanelEntry>;
 }
 
-interface CommonOverviewPropsWithToolbar {
-  openTimeline: (panelUuid: string) => void;
-  openTrendline: (panelUuid: string, testUuid: string) => void;
-}
-
-interface CommonOverviewPropsWithoutToolbar {
-  hideToolbar: true;
-}
-
-type Only<T, U> = {
-  [P in keyof T]: T[P];
-} & {
-  [P in keyof Omit<U, keyof T>]?: never;
-};
-
-type Either<T, U> = Only<T, U> | Only<U, T>;
-
-type CommonOverviewProps = CommonOverviewPropsBase &
-  Either<CommonOverviewPropsWithToolbar, CommonOverviewPropsWithoutToolbar>;
-
-const CommonOverview: React.FC<CommonOverviewProps> = ({
-  overviewData = [],
-  openTimeline,
-  openTrendline,
-  insertSeparator = false,
-  hideToolbar = false,
-  patientUuid,
-}) => {
+const CommonOverview: React.FC<CommonOverviewProps> = ({ patientUuid, overviewData = [] }) => {
```

Comment:
The only usage of CommonOverview is the ExternalOverview component. So I got rid of everything that wasn't being used by that. CommonOverview could probably just be refactored into ExternalOverview, but I haven't done that.